### ISSUE TITLE
Remove global variables?

### ISSUE THREAD

**Issue by ralfstrandell** [jariarkko/psgeo#323]
We could move global variables into the cfg object (which is passed as a parameter to about everything).

**Comment by ralfstrandell** [comment]
Global variables should be referenced like this nowadays:
globalThis.variable
But let's just move the bulk of these into psgeoui.js once refactoring is done.

**Comment by ralfstrandell** [comment]
Nah. psgeo.js will be split into psgeo and psgeoui this spring. Once this is done, most of the "global" variables can be found inside an ui object. Most of the currently global vars are also not config by type, but more program state. Hence, I would skip the cfg.variable approach, too. Wontfix.
